### PR TITLE
Karinamzalez/1083 poa info case details

### PR DESCRIPTION
**Pull request by karinamzalez**
Resolves https://vajira.max.gov/browse/CASEFLOW-1083

### Description
- update copy for NO POA and add update corresponding spec.  
- add poa_last_synced_at to unrecognized_power_of_attorney
- update Attorney details title

BEFORE|AFTER

Screen Shot 2021-04-14 at 12 43 10 PM
| 
Screen Shot 2021-04-14 at 12 45 39 PM

### CODE REVIEW

File: spec/feature/queue/case_details_spec.rb [hunk 1]
```diff
@@ -334,6 +334,25 @@
...
         end
       end
+
+      context "when an unrecognized appellant doesn't have a POA" do
+        let!(:claimant) do
+          create(
+            :claimant,
+            unrecognized_appellant: ua,
+            decision_review: appeal,
+            type: "OtherClaimant",
```
Comment (nanotone) [inline spec/feature/queue/case_details_spec.rb:344]: CodeClimate is complaining about the trailing comma here.

File: client/test/app/queue/docketSwitch/RecommendDocketSwitchContainer.test.js [hunk 2]
```diff
@@ -63,18 +63,18 @@ describe('formatDocketSwitchRecommendation', () => {
...
-        hyperlink,
-        disposition,
-      });
+  //   it('properly formats', () => {
+  //     const res = formatDocketSwitchRecommendation({
+  //       summary,
+  //       timely,
+  //       hyperlink,
+  //       disposition,
+  //     });
```
Comment (nanotone) [inline client/test/app/queue/docketSwitch/RecommendDocketSwitchContainer.test.js:77]: I think this was unintentionally left in?

**Review by nanotone — APPROVED** [review]
Couple minor issues but other than that it looks good!

File: client/app/queue/PowerOfAttorneyDetail.jsx [hunk 3]
```diff
@@ -64,7 +64,7 @@ const PowerOfAttorneyDetailWrapper = (WrappedComponent) => {
 
     return hasPowerOfAttorneyDetails ?
       <WrappedComponent powerOfAttorney={powerOfAttorney} /> :
-      <p><em>{COPY.CASE_DETAILS_NO_POA}</em></p>;
+      <p {...noPOA}><em>{COPY.CASE_DETAILS_NO_POA}</em></p>;
```
Comment (nanotone) [inline client/app/queue/PowerOfAttorneyDetail.jsx:67]: After going back and forth on this, I actually think the extra styling isn't needed. The grey color might be a side-effect of the Jira screenshot, but the actual color should match Caseflow Gray-Dark, as it does for the Veteran box:


And the `<em>` is what takes care of the italicizing.

File: client/test/app/queue/docketSwitch/RecommendDocketSwitchContainer.test.js (hunk 2 above)
Comment (karinamzalez) [reply client/test/app/queue/docketSwitch/RecommendDocketSwitchContainer.test.js:77]: It was -- I took it out but forgot to push the changes up 😅

**Pull request merged**